applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -25,6 +25,7 @@
 #include "le_audio.h"
 #include "audio_datapath.h"
 #include "audio_sync_timer.h"
+#include "sw_codec_select.h"
```

Comment:
We want to have as few dependencies as possible. This is related to another discussion below.